docs/burger-discounts/index.md

```diff
@@ -407,21 +407,21 @@ using lists, which are a better fit for this problem.
 - Type inference is less effective inside functions that don't call other
   functions. In those cases, you can give the compiler more information:
   - Type annotate the function arguments
-  - Use the full name for value
+  - Use the full name for a value used inside the function
 - The `Stdlib` module is opened by default
 - Records are immutable
-- Use record copy syntax to make copies of records that have different values
-  for some fields
+- Use record copy syntax to copy a record while changing some fields on the
+  copy, e.g. `{...burger, lettuce: true, onions: 3}`
 - OCaml doesn't allow you to ignore the return value of functions (unless the
   value is `()`), so you can use `Stdlib.ignore` to explicitly discard return
   values
 - Runtime representations of common data types:
-  - Variant constructor without argument -> integer
-  - Variant constructor with argument -> JavaScript object
-  - Record -> JavaScript object
-  - Array -> JavaScript array
-  - `None` -> `undefined`
-  - `Some(value)` -> `value`
+  - Variant constructor without argument ⮕ integer
+  - Variant constructor with argument ⮕ JavaScript object
+  - Record ⮕ JavaScript object
+  - Array ⮕ JavaScript array
+  - `None` ⮕ `undefined`
+  - `Some(value)` ⮕ `value`
```

Comment:
What about these? a bit slimmer:

```suggestion
  - Record → JavaScript object
  - Array → JavaScript array
  - `None` → `undefined`
  - `Some(value)` → `value`
```

Reply:
Thanks! They don't look good in the editor but these thinner arrows do look better on the actual page. Updated in https://github.com/melange-re/melange-for-react-devs/pull/36/commits/8f289acb228832aa0e626eb8344a194c466e0057